src/Security/Authentication/JwtBearer/src/JwtBearerOptions.cs

```diff
@@ -103,8 +113,14 @@ public new JwtBearerEvents Events
     /// <summary>
     /// Gets the ordered list of <see cref="ISecurityTokenValidator"/> used to validate access tokens.
     /// </summary>
+    [Obsolete("SecurityTokenValidators is no longer used by default. Use TokenHandlers instead. To continue using SecurityTokenValidators, set UseSecurityTokenValidators to true. See https://aka.ms/aspnetcore8/security-token-changes")]
```

Comment:
@Tratcher @halter73 @captainsafia - what do you think about defining a new `DiagnosticId` for this obsoletion? That way people who are using SecurityTokenValidators, and want to keep using it, don't need to suppress `CS0618` (the catch-all obsolete ID), and instead can just globally suppress the specific `ASP1234` ID. Then we could do the same in our tests and not need to `#pragma warning disable` everywhere.

Reply:
Let’s do this for rc1. @keegan-caruso - can you log a follow up issue for it?

Reply:
@eerhardt https://github.com/dotnet/aspnetcore/issues/49558